ckanext/geodatagov/cli.py

```diff
@@ -404,6 +404,8 @@ def db_solr_sync(dryrun, cleanup_solr, update_solr):
 
     # now it is cleaned, change dict back to a set.
     active_package = {(k,) + cleaning[k] for k in cleaning}
```

Comment:
what does this mean `(k,) + cleaning[k]`?

Reply:
I added comments in the code to clarify the purpose of this line.

Reply:
thanks!